docs/guides/index.md

```diff
@@ -527,15 +527,16 @@ Events][animatingonevents] below.
 
 In JavaScript, we grab the box with [`querySelector`][queryselector], use
 [`addEventListener`][addeventlistener], and then [`setAttribute`][setattribute]
-to make the box grow its scale when its hovered over. Note that A-Frame adds
+to make the box grow its scale when its hovered over.The code must appear after
+the scene so that the box has loaded when the script runs. Note that A-Frame adds
```

Comment:
I meant. Two PRs: one with only the typo fix and the other with the doc clarification